lab_2_retrieval_w_bm25/main.py

```diff
@@ -18,6 +18,22 @@ def tokenize(text: str) -> list[str] | None:
 
     In case of corrupt input arguments, None is returned.
     """
+    if not isinstance(text, str):
+        return None
+
+    if any(symb.isalpha() for symb in text) is False:
```

Comment:
Is this check really necessary?

Reply:
wow, I was sure the code won't work without it but it works. Done